src/main/java/com/unascribed/fabrication/client/SpriteLava.java

```diff
@@ -52,7 +52,7 @@ public void tickAnimation() {
 					}
 				}
 
-				this.field_76878_h[var1 + var2 * 16] = var3 / 10.0F + (this.field_76879_i[(var1 + 0 & 15) + (var2 + 0 & 15) * 16] + this.field_76879_i[(var1 + 1 & 15) + (var2 + 0 & 15) * 16] + this.field_76879_i[(var1 + 1 & 15) + (var2 + 1 & 15) * 16] + this.field_76879_i[(var1 + 0 & 15) + (var2 + 1 & 15) * 16]) / 4.0F * 0.8F;
+				this.field_76878_h[var1 + var2 * 16] = var3 / 10.0F + (this.field_76879_i[(var1 & 15) + (var2 & 15) * 16] + this.field_76879_i[(var1 + 1 & 15) + (var2 & 15) * 16] + this.field_76879_i[(var1 + 1 & 15) + (var2 + 1 & 15) * 16] + this.field_76879_i[(var1 & 15) + (var2 + 1 & 15) * 16]) / 4.0F * 0.8F;
```

Comment:
These `+ 0`s come from obfuscated code; I forget the exact reason they exist. Rather than just remove these I'd prefer to finish deobfuscating this code.